dandiapi/api/models/zarr.py

```diff
@@ -312,7 +336,19 @@ class Meta(BaseZarrArchive.Meta):
             models.UniqueConstraint(
                 name='unique-embargo-dandiset-name',
                 fields=['dandiset', 'name'],
-            )
+            ),
+            models.CheckConstraint(
+                name='nonempty-embargo-checksum',
+                check=models.Q(checksum__length__gt=0),
```

Comment:
Maybe a comment here to clarify where `__length` comes from? Something like
```suggestion
				# Use custom `length` lookup (defined previously in this file) in the constraint
                check=models.Q(checksum__length__gt=0),
```